### ISSUE TITLE
Unclear Hex description

### ISSUE THREAD

**Issue by henrik** [rrrene/inch_ex#12]
Could mention what it does and not just what it ports :)

**Comment by rrrene** [comment]
Good point. What about "Provides a Mix task that gives you hints where to improve your inline docs."?

**Comment by henrik** [comment]
Sounds good!
